packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -53,7 +53,7 @@ export type NormalizeObjectNodeFields<T extends FlexObjectNodeFields> = {
  * These extends constraints only serve as documentation:
```

Comment:
Nit: might be nice to link to TS docs for the mention of `extends constraints` above.

Also we should be sure to use `{@link }` syntax instead of Markdown link syntax in our API docs, at least for now. Markdown links are not in TSDoc's spec (at least not last I checked).